packages/react/src/experimental/RichText/CoreEditor/Extensions/MoodTracker/index.tsx

```diff
@@ -114,49 +170,55 @@ export const MoodTrackerView: React.FC<NodeViewProps> = ({ node }) => {
                 transformOrigin: "top",
                 overflow: "hidden",
               }}
-              className="pt-3"
             >
-              <div className="flex flex-col gap-2">
-                {data.days.map((day, index) => {
-                  return (
-                    <motion.div
-                      key={index}
-                      initial={{
-                        opacity: 0,
-                        y: -10,
-                        scale: 0.95,
-                      }}
-                      animate={{
-                        opacity: 1,
-                        y: 0,
-                        scale: 1,
-                      }}
-                      transition={{
-                        delay: index * 0.05,
-                        duration: 0.2,
-                        ease: "easeOut",
-                      }}
-                      className="flex flex-row items-center gap-2"
-                    >
-                      <div className="flex items-center justify-center rounded-full">
-                        <Icon
-                          icon={pulseIcon[day.mood]}
-                          size="lg"
-                          className={pulseIconStyle({ pulse: day.mood })}
-                        />
-                      </div>
-                      <p className="text-f1-text-primary text-md font-normal">
-                        <span className="font-semibold">{day.day}:</span>{" "}
-                        {day.comment}
-                      </p>
-                    </motion.div>
-                  )
-                })}
-              </div>
+              <motion.div
+                className="text-f1-text-primary"
+                initial={{ opacity: 0, y: -10 }}
+                animate={{ opacity: 1, y: 0 }}
+                transition={{ delay: 0.1, duration: 0.3 }}
+              >
+                <div className="flex flex-col gap-2">
+                  {data.days.map((day, index) => {
+                    return (
+                      <motion.div
```

Comment:
As we're not doing anything else here we could get rid of the `return`

```suggestion
                  {data.days.map((day, index) => (
                      <motion.div
```